homework-autocomplete-ui/src/components/search-component/search-component.js

```diff
@@ -0,0 +1,12 @@
+import createInput from './utils/createInput';
+import createResultBlock from './utils/createResultBlock';
+import addInputListeners from './utils/addInputListeners';
```

Comment:
`utils` directory used for some helper function (f.e. find in array, and others). These functions should be in a directory like `presenatation`, `view` or something like this.